src/libmongoc/tests/test-mongoc-structured-log.c

```diff
@@ -142,6 +143,7 @@ test_structured_log_plain (void)
 
    mongoc_structured_log_opts_t *opts = mongoc_structured_log_opts_new ();
    mongoc_structured_log_opts_set_handler (opts, structured_log_func, &assumption);
+   ASSERT (mongoc_structured_log_opts_set_max_document_length (opts, 10000));
```

Comment:
```suggestion
   ASSERT (
      mongoc_structured_log_opts_set_max_document_length (opts, MONGOC_STRUCTURED_LOG_DEFAULT_MAX_DOCUMENT_LENGTH));
```
Should these values of `10000` be `MONGOC_STRUCTURED_LOG_DEFAULT_MAX_DOCUMENT_LENGTH` (1000) instead?

Reply:
The specific value doesn't really matter as long as it's more than the small lengths used in the test, but my intent was to follow the advice from the CLAM spec:

"Drivers MUST run the logging tests with their max document
length setting (as described in the [logging specification](../../logging/logging.md#configurable-max-document-length))
set to a large value e.g. 10,000; this is necessary in order for the driver to emit the full server reply (and to allow
matching against that reply) on certain MongoDB versions and topologies."

It's mostly arbitrary but I'll add a comment explaining the choice just so it doesn't seem out of place.